scss/typography/_dropcaps.scss

```diff
@@ -2,13 +2,30 @@
 /// @group typography-dropcaps
 ////
 
+/// let 1em = 16px for the conversion tool
+/// @type Number
+$dropcaps-browser-context: 16;
+
+/// Safe px to em conversion
+@function em($pixels, $context: $dropcaps-browser-context) {
+  @if (unitless($pixels)) {
+    $pixels: $pixels * 1px;
+  }
+
+  @if (unitless($context)) {
```

Comment:
You can use `@func unitless-calc()`. So `$dropcaps-browser-context` is not necessary (it is assumed inside `unitless-calc`).